tests/sentry/integrations/opsgenie/test_integration.py

```diff
@@ -122,10 +120,26 @@ def test_update_config_invalid(self):
 
         integration.add_organization(self.organization, self.user)
         installation = integration.get_installation(self.organization.id)
-        opsgenie_client = installation.get_client(integration_key="1234-bad")
 
-        data = {"team_table": [{"id": "", "team": "cool-team", "integration_key": "1234-bad"}]}
-        responses.add(responses.GET, url=f"{opsgenie_client.base_url}/teams/cool-team")
+        org_integration = OrganizationIntegration.objects.get(integration_id=integration.id)
+        team_id = str(org_integration.id) + "-" + "cool-team"
+
+        # valid
+        data = {"team_table": [{"id": "", "team": "cool-team", "integration_key": "1234"}]}
```

Comment:
should we move this into its own test? i think this is now testing both invalid and valid updates

Reply:
I'm just adding a team to config here (tested previously). The actual test is adding a duplicate team.